content/blog/wingingit/winging-it-05.md

```diff
@@ -1,22 +1,23 @@
 ---
-eleventyExcludeFromCollections: true
-permalink: false
-title: FastAPI
+title: Streamline Your API Development with FastAPI
 episode: 5
 tags:
   - FastAPI
+  - Python
+  - SQLAlchemy
 author:
   - ed
-date: 2023-12-21
-length: 30 mins
+  - davidh
+  - james
+date: 2024-03-19
```

Comment:
This too will need an update

Reply:
@stacyk So sorry! I had a bunch of those things that you commented on already update, but realized (shortly) after your review) that I hadn't pushed them.